core/com.b2international.snowowl.datastore.server/src/com/b2international/snowowl/datastore/server/internal/CDOBasedRepository.java

```diff
@@ -97,11 +101,13 @@
  */
 public final class CDOBasedRepository extends DelegatingServiceProvider implements InternalRepository, CDOCommitInfoHandler {
 
-	private static final String REINDEX_KEY = "snowowl.reindex";
+	private static final Logger LOG = LoggerFactory.getLogger(CDOBasedRepository.class);
```

Comment:
Use `LoggerFactory.getLogger("repository")` instead of the implementation class.